WalletWasabi.Gui/Shell/MainMenu/FileMainMenuItems.cs

```diff
@@ -43,7 +43,7 @@ public FileMainMenuItems(IMenuItemFactory menuItemFactory)
 
 		#region MenuItem
 
-		[ExportMainMenuItem("File", "Generate Wallet")]
+		[ExportMainMenuItem("File", "Generate New Wallet")]
```

Comment:
NACK, it makes it longer, the spelling is correct and the function is not ambiguous.